docs/tags.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+---
+
+# Tags
+
+{% assign tags_and_posts = site.tags | sort %}
+{% for tag_and_posts in tags_and_posts %}
+  {% assign tag = tag_and_posts | first | slugify %}
+  {% assign posts = tag_and_posts | last %}
```

Comment:
:question: Are posts given in any pre-sorted order?

Reply:
I can't find anything in the Jekyll docs, but from looking at another blog which doesn't seem to be explicitly ordering anything, it does seem to be reverse-ordered by date (i.e. newest first, oldest last).  I'll leave this for now, but it could easily be fixed with `... | sort: "date" | reverse`.